ZenPacks/zenoss/Microsoft/Windows/modeler/plugins/zenoss/winrm/HardDisks.py

```diff
@@ -128,7 +136,9 @@ def make_disk_ids(disk_drive):
     disk_ids.append(pnpDevice)
 
     # Append SerialNumber
-    serial = disk_drive.SerialNumber.strip() if hasattr(disk_drive, 'SerialNumber') else None
+    serial = None
+    if hasattr(disk_drive, 'SerialNumber'):
+        serial = disk_drive.SerialNumber.strip()
```

Comment:
it can't be because if it is set to None initially, I change it to '' before it comes into this function